addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1725,10 +1725,12 @@ MockServer.include({
                 return {
                     id: partner.id,
                     im_status: user.im_status || 'offline',
+                    email: partner.email,
                     name: partner.name,
                     user_id: user.id,
                 };
-            });
+            }).sort((a, b) => (a - b))
+              .sort((a, b) => (a.name > b.name) ? 1 : -1);
```

Comment:
I don't think that works, one `sort` will probably not be stable compared to the other.
Both conditions should be part of only one sort method: if name is equeal, return the condition on id.
Note that it would be better to compare ids instead of objects (I have no idea what difference between objects like you did it does).

Reply:
My bad, I want to do 
```js
.sort((a, b) => (a.id - b.id))
.sort((a, b) => (a.name > b.name) ? 1 : -1);
```